App_Use_Cases.tex

```diff
@@ -502,7 +508,85 @@ \subsection{Use Case Details}
 
 \littleheader{Related Constants}
 
-\refconst{PMIX_SUCCESS}
+\refconst{PMIX_SUCCESS}\\
 \refconst{PMIX_ERR_NOT_SUPPORTED}
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Cross-Version Compatability}
+\label{app:uc-cross-version}
+
+\subsection{Use Case Summary}
+
+The \ac{PMIx} interface serves as a conduit between clients (e.g., \ac{MPI} libraries), tools (e.g., debuggers), and servers (e.g., \acp{RM}).
+As such, it is probable that a process operating in one of these roles (e.g., as a client or tool) is running a different version of the same \ac{PMIx} implementation than the process with which it is communicating that is operating in a different role (e.g., as a server).
+For processes running in containers cross-version compatibility is especially important because the container image and the system software levels will naturally evolve and drift apart.
+As such, there is a need for the \ac{PMIx} implementation to provide cross-version compatibility.
+
+The responsibility for providing cross-version compatibility is a feature of the \ac{PMIx} implementation and not necessarily of the \ac{PMIx} standard.
+The \ac{PMIx} standard must strive to enable, and never limit, both the cross-version compatibility in the \ac{PMIx} implementation, and the ability for a \ac{PMIx} consumer to adapt to cross-version differences in capabilities.
+
+This use case is focused on cross-version compatibility between versions of the same \ac{PMIx} implementation and not between different \ac{PMIx} implementations.
+
+Cross-version compatibility responsibilities are not restricted to \ac{PMIx}, but a general issue for any library that coordinates across multiple processes. This includes, but not limited to, client/server libraries, and libraries with a user-space and kernel-space component (e.g., high-performance interconnect libraries).
```

Comment:
Maybe hoist this up into the opening paragraph?  Seems like this fits in well with the more general opening paragraph that sets the context.  No strong feelings though, fine if it stays here.

Reply:
I'll leave it here for now and see what others think. I'm inclined to leave it where it is as a closing statement for generality.